src/TestSuite/StringCompareTrait.php

```diff
@@ -43,6 +54,15 @@ public function assertSameAsFile($path, $result)
     {
         $path = $this->_compareBasePath . $path;
```

Comment:
I don't see using absolute paths as a valid use case - I mean, if you wanted to use it like that just don't set the `compareBasePath` to anything.

However, since there have been further comments, if the assertSameAsFile argument points directly at a file I've disabled the use of compareBasePath - so you can use an absolute path if you want.